kafka-source/src/main/java/org/springframework/cloud/stream/module/kafka/source/KafkaConfigurationProperties.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.kafka.source;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Properties related to connecting to Kafka broker.
+ *
+ * @author Soby Chacko
+ */
+@ConfigurationProperties
+public class KafkaConfigurationProperties {
+
+	private String zkConnect = "localhost:2181";
+
+	private String zkConnectionTimeout = "6000";
+
+	private String zkSessionTimeout = "6000";
+
+	private String zkSyncTime = "2000";
+
+	private String keyDecoder;
+
+	private String payloadDecoder;
```

Comment:
I see that these are converted to the `Class` in the `KafkaSource` thereafter.
The `private Class payloadDecoder;` works well here in the properties.